Application/src/main/java/com/kenzie/appserver/service/DrinkService.java

```diff
@@ -58,36 +57,15 @@ public Drink getDrink(String name, String userId){
 
     //add drink
     public Drink addDrink(Drink request) {
-        LambdaDrink drinkExists = null;
-        try {
-            drinkExists = lambdaServiceClient.getDrink(request.getId());
-        } catch (IllegalArgumentException e){{
-            // do nothing
-        }}
-
-        if(drinkExists != null) {
-            throw new UserHasExistingDrinkException("Drink should be updated instead");
-        }
-
         LambdaDrink drink = new LambdaDrink(request.getId(), request.getName(), request.getIngredients(), request.getUserId());
```

Comment:
the class Drink does not have null or final checks so this could throw an exception

Reply:
Null checking is done at controller level prior to service